src/View/SSViewer_Scope.php

```diff
@@ -368,4 +377,14 @@ protected function getUpIndex()
     {
         return $this->upIndex;
     }
+
+    private function convertScalarToDBField(bool|string|float|int|null $value): DBField
```

Comment:
```suggestion
    private function convertScalarToDBField(bool|string|float|int $value): DBField
```
Sorry, should have specifically requested this at the same time as https://github.com/silverstripe/silverstripe-framework/pull/11238#discussion_r1600761107

Reply:
Done